tsconfig.json

```
@@ -0,0 +1,15 @@
+{
+  "compilerOptions": {
+    "module": "ES2020",
+    "lib": ["dom", "es5"],
```

Shouldn't that lib be es2020?

```suggestion
    "lib": ["dom", "es2020"],
```

---

I think best is `es6`, because `es2020` features currently not used in this code and `es5` may lead to some excessive code when transpiling to `es6` and higher.